handler.go

```diff
@@ -0,0 +1,20 @@
+package microerror
+
+type Handler interface {
+	// Cause returns the cause of the given error. If the cause of the err can not
+	// be found it returns the err itself.
+	//
+	// Cause is the usual way to diagnose errors that may have been wrapped by Mask
+	// or Maskf.
+	Cause(err error) error
+
+	// Mask is a simple error masker. Masked errors act as tracers within the
+	// source code. Inspecting an masked error shows where the error was passed
+	// through within the code base. This is gold for debugging and bug hunting.
+	Mask(err error) error
+
+	// Maskf is like Mask. In addition to that it takes a format string and
+	// variadic arguments like fmt.Sprintf. The format string and variadic
+	// arguments are used to annotate the given errgo error.
+	Maskf(err error, f string, v ...interface{}) error
```

Comment:
I removed `Any` inside the functions as it adds only clutter. There are still `MaskAny` aliases in `error.go`.